addons/test_mail_full/tests/test_sms_post.py

```diff
@@ -338,6 +338,26 @@ def test_message_sms_crash_wrong_number_single(self):
             {'partner': self.partner_3, 'state': 'sent'},
         ], self._test_body, messages)
 
+    def test_message_sms_crash_unregistered(self):
```

Comment:
Linting: I tried to organize a bit methods by alphabetical order, because tests are done by that order (easier when tests crash in batch :D ). Maybe relocate after test_message_sms_crash_server_crash .